### DISCUSSION TITLE
Interface Session-help panel

### DISCUSSION THREAD

**Discussion by gerhardriener** [posit-dev/positron#4935] [Feedback]
In the Session panel on the right hand side there are two lines indicating whether one is in session, help viewer or connections (see screenshot)
![Capture](https://github.com/user-attachments/assets/366aca4c-cfb7-4c10-b983-2e46b56f76bb)

Wouldn't it save space an make the interface less cluttered when removing the second line?

**Comment by juliasilge** [comment]
I believe this is the same as https://github.com/posit-dev/positron/issues/4427. Want to take a look and see if you have additional info to add?

**Reply by gerhardriener** [reply]
Yes, that's effectively a duplicate.
